docs/market_analysis.rst

```diff
@@ -0,0 +1,291 @@
+.. _market_analysis-section:
+
+Market Analysis
+---------------
+
+Overview
+``````````
+This feature records a currency's market and allows the bot see trends. With this data, we can compute a recommended minimum lending rate per currency to avoid lending at times when the rate dips.
+
+When this module is enabled it will start recording the lending rates for the market in an sqlite database. This will be seen in the market_data folder for your bot. This supersedes the previous method of storing it in a file. The files can be removed if you have them from older versions of the bot.
+
+There will be a DB created for each currency you wish to record. These can be enabled in the `analyseCurrencies`_ configuration option.  
+
+ .. warning:: The more currencies you record, the more data stored on disk and CPU processing time will be used. You will also not get as frequent results for the currencies, i.e. You may have trouble getting results for your configured ``analyseUpdateInterval`` This is explained further in the `Recording currencies`_ section. 
+
+A quick list of each config option and what they do
+
+========================= =============================================================================================
+`analyseCurrencies`_      A list of each currency you wish to record and analyse
+`analyseUpdateInterval`_  The frequency between rates requested and stored in the DB
+`lendingStyle`_           The percentage used for the percentile calculation
+`percentile_seconds`_     The number of seconds to analyse when working out the percentile
+`MACD_long_win_seconds`_  The number of seconds to used for the long moving average
+`MACD_short_win_seconds`_ The number of seconds to used for the short moving average
+`keep_history_seconds`_   The age (in seconds) of the oldest data you wish to keep in the DB
+`recorded_levels`_        The depth of the lending book to record in the DB, i.e. how many unfilled loans
+`data_tolerance`_         The percentage of data that can be ignore as missing for the time requested in
+                          ``percentile_seconds`` and ``MACD_long_win_seconds``
+`daily_min_method`_       Which method (MACD or percentile) to use for the daily min calculation
+`MACD_multiplier`_        Only valid for MACD method. The figure to scale up the returned rate value from the MACD calculation
```

Comment:
Yes, I agree. But I'd really like to do that all in a big config fix that's really overdue. 

https://github.com/BitBotFactory/poloniexlendingbot/issues/364